_episodes/l2-01-managing_contributions.md

```diff
@@ -61,6 +59,12 @@ Pull requests can be created by visiting the `Pull request` tab in the repositor
 > 1. Create a remote repository with your recipe repository.
 > 2. Create a new branch with some changes and push the branch to the remote repository.
 > 3. Create a pull request with a suitable title and description to merge the branch containing your changes into the main branch.
+> 
+>> ### Note
```

Comment:
For some reason the level 3 heading `###` added lots of whitespace above the heading when in this callout block. It looks better with a level 4 `####` heading.
```suggestion
>> #### Note
```